apps/iotune/iotune.cc

```diff
@@ -138,6 +141,10 @@ struct evaluation_directory {
                 _min_data_transfer_size = std::max(_min_data_transfer_size, disk_min_io_size);
                 _max_iodepth += read_first_line_as<uint64_t>(queue_dir / "nr_requests");
                 _disks_per_array++;
+
+                _physical_block_size = read_first_line_as<uint64_t>(queue_dir / "physical_block_size");
```

Comment:
What if sysfs file is missing?

Reply:
Adjusted the PR according to the offline discussion. Instead of querying values I added a parameter that allows users to supply the desired buffer size for measurements of random read and random write IOPS.